build-s2i.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
as said, a good alternative would be a Makefile. See https://github.com/AICoE/prometheus-flatliner/blob/master/Makefile for a cool one that also reads in `.env` - this way you don't have to specify the 👇 variables in that shell script here.

Reply:
Looks good. I can incorporate that here.

Reply:
This file has been removed.